packages/@expo/cli/src/start/server/middleware/ManifestMiddleware.ts

```diff
@@ -164,6 +164,18 @@ export abstract class ManifestMiddleware<
   /** Parse request headers into options. */
   public abstract getParsedHeaders(req: ServerRequest): TManifestRequestInfo;
 
+  /**
+   * This header is specified as a string "true" or "false", in one of two headers:
+   * - exponent-accept-signature
+   * - expo-accept-signature
+   */
+  protected getLegacyAcceptSignatureHeader(req: ServerRequest): boolean {
+    return (
+      req.headers['exponent-accept-signature'] === 'true' ||
+      req.headers['expo-accept-signature'] === 'true'
```

Comment:
Use a case-insensitive API to get the headers:
```suggestion
      req.get('exponent-accept-signature') === 'true' ||
      req.get('expo-accept-signature') === 'true'
```

I'm not sure what's going on with the TS types but assuming `req` is an Express request object, it will have a `get(headerName)` method that is case-insensitive.

Reply:
It is typed as `express.Request | http.IncomingMessage` so it can't use `get` unfortunately.